doc/classes/SpinBox.xml

```diff
@@ -56,15 +56,22 @@
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable" default="true" keywords="readonly, disabled, enabled">
 			If [code]true[/code], the [SpinBox] will be editable. Otherwise, it will be read only.
 		</member>
-		<member name="prefix" type="String" setter="set_prefix" getter="get_prefix" default="&quot;&quot;">
+		<member name="format" type="String" setter="set_format" getter="get_format" default="&quot;&quot;">
+			Specifies formatting of the displayed numeric value, using the same rules as operator [code]%[/code] in [String]. Only applies when the value is not being currently edited. If empty, the value will use default display. Using [code]%s[/code] placeholder will display the value with default formatting. If the format is invalid, the [SpinBox] will show "ERROR", with the actual error message available in the configuration warning.
```

Comment:
I swear to God `[operator String.operator %]` **should** be a valid reference based on what I have seen of the codebase, and yet it may not be for some reason.